src/main/java/practice/StreamPractice.java

```diff
@@ -1,20 +1,40 @@
 package practice;
 
-import java.util.Collections;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+    private CandidateValidator candidateValidator = new CandidateValidator();
+
+    public static void main(String[] args) {
+        List<String> list = new ArrayList<>();
+        list.add("2,4");
+        list.add("7,9");
+        list.add("10");
+        findMinEvenNumber(list);
+    }
```

Comment:
```suggestion
    public static void main(String[] args) {
        List<String> list = new ArrayList<>();
        list.add("2,4");
        list.add("7,9");
        list.add("10");
        findMinEvenNumber(list);
    }
```

remove this, this task is tested by unit testing